addons/area_markers/XEH_postInit.sqf

```diff
@@ -0,0 +1,84 @@
+#include "script_component.hpp"
+
+if (isServer) then {
```

Comment:
Perhaps this could be extracted into a separate file and add a doc block as this is quite long?

Reply:
I don't see what that would do, Just split up code when `XEH_postInit` is generally the first place to look for added CBA events. Also, I think the code is simple enough and doesn't need a doc block.